### PR TITLE
Bay update, now Travis approved

### PR DESCRIPTION
**Pull request by caelaislinn**
Maybe this will fix the unit test issues

### CODE REVIEW

**Comment by bloxgate** [comment]
Seems to break a few things with our new chemicals, and overrides some CI changes we'd made regarding file checksums.

**Comment by caelaislinn** [comment]
Travis approved, mergable.

File: code/modules/halo/chemicals/medicine.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 /datum/reagent/triadrenaline
 	name = "Tri-Adrenaline"
-	id = "triadrenaline"
+	//id = "triadrenaline"
```
Comment (bloxgate) [inline code/modules/halo/chemicals/medicine.dm]: I'm not too familiar with the recent chemical changes, but is it still possible to make these without the ID?

Comment (caelaislinn) [reply code/modules/halo/chemicals/medicine.dm]: It would have disabled the reaction, but I've updated it to work now.

**Pull request merged**